contracts/taskModules/ProxyModule.sol

```diff
@@ -31,19 +30,75 @@ contract ProxyModule is TaskModuleBase {
         _deployIfNoProxy(_taskCreator);
     }
 
+    /**
+     * @inheritdoc TaskModuleBase
+     * @dev _taskCreator cannot create task to other user's proxy
+     */
+    function preCreateTask(address _taskCreator, address _execAddress)
+        external
+        view
+        override
+        returns (address, address)
+    {
+        bool isExecAddressProxy = opsProxyFactory.isProxy(_execAddress);
+
+        if (isExecAddressProxy) {
+            address ownerOfExecAddress = opsProxyFactory.getOwnerOf(
+                _execAddress
+            );
+            require(
+                _taskCreator == ownerOfExecAddress,
+                "ProxyModule: Only owner of proxy"
+            );
```

Comment:
this would fail in the case that I'm both creating the task through my proxy & using the proxy as the execAddress
ie: _taskCreator = proxyAddress  && _execAdddress = proxyAddress